services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/ApplicationConfigurationV2PublicationConfig.java

```diff
@@ -38,35 +43,77 @@
 import app.coronawarn.server.services.distribution.config.DistributionServiceConfig.AppConfigParameters.IosPrivacyPreservingAnalyticsParameters;
 import java.util.List;
 import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 /**
- * Provides the application configuration needed for mobile clients which use Exposure Notification
- * Framework V2. As of CWA version 1.7, Android and IOS configurations have been separated and
- * distributed in CDNs at different URLs. This Spring bean loads the default values defined in the
- * YAML configurations of each device type found under <code> /main-config/ </code> folder,
- * extends it with some Distribution Service global parameters and registers them for usage during
- * file archiving and bundling.
+ * Provides the application configuration needed for mobile clients which use Exposure Notification Framework V2. As of
+ * CWA version 1.7, Android and IOS configurations have been separated and distributed in CDNs at different URLs. This
+ * Spring bean loads the default values defined in the YAML configurations of each device type found under <code>
+ * /main-config/ </code> folder, extends it with some Distribution Service global parameters and registers them for
+ * usage during file archiving and bundling.
  */
 @Configuration
 public class ApplicationConfigurationV2PublicationConfig {
 
   /**
    * The location of the exposure configuration source files for Android and Ios.
    */
-  private static final String V2_RISK_PARAMETERS_FILE = "main-config/v2/risk-calculation-parameters.yaml";
+  private static final String V1_RISK_PARAMETERS_FILE = "main-config/v2/risk-calculation-parameters.yaml";
+  private static final String V2_RISK_PARAMETERS_FILE = "main-config/v2/risk-calculation-parameters-v2.yaml";
```

Comment:
I find it a bit confusing that the path contains `v2` twice (although I understand why). How about changing the filename to `risk-calculation-parameters-1.15.yaml`? Or we change the `v2` in between to `v3` and host the app config on CDN also under `v3` instead of `v2`?

Reply:
I will change the filename to risk-calculation-parameters-1.15.yaml